lib/src/components/NxModal/NxModal.tsx

```diff
@@ -4,147 +4,41 @@
-          onCancel(new Event('cancel', { cancelable: true }));
-        }
-      }
-    }
-  }
-
-  useEffect(function() {
-    /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
-    const el = dialogRef.current!,
-
-        // HTML <dialog> elements are supposed to remember what element was focused before they were opened,
-        // and restore focus to that element when they are closed. No browsers appear to implement this currently,
-        // so we do it ourselves
-        previouslyFocusedEl = document.activeElement;
-
-    setDialogRefState(el);
-
-    /*
-     * This will cause the document to become "blocked by the modal dialog"
-     * (https://html.spec.whatwg.org/multipage/interaction.html#blocked-by-a-modal-dialog)
-     * meaning that only the modal and its contents are interactable/focusable.
-     *
-     * Note: not supported in safari, which is why the conditional is here. Once safari gets
-     * it together, we should be able to take advantage of the "top layer" functionality built into
-     * the browser around modals to simplify the NxModal styling around z-index handling
-     */
-    if (hasNativeModalSupport) {
-      // https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1029#issuecomment-968299542
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      (el as any).showModal();
-    }
-    else {
-      // without native support we don't trap focus in the modal, but we can at least start it off there
-      el.focus();
-    }
-
-    return () => {
-      if (previouslyFocusedEl && previouslyFocusedEl instanceof HTMLElement) {
-        // The useEffect cleanup executes while the modal is still present (in React 16 at least). While the modal
-        // still exists, the document is still "blocked by the modal dialog" so trying to focus elements outside of
-        // it won't work. So we have to wait until the next cycle of the event loop when it's gone
-        Promise.resolve().then(() => previouslyFocusedEl.focus());
-      }
-    };
-  }, []);
-
-  // listen to the native HTMLDialogElement cancel event which supporting browsers fire when the modal is closed
-  // via ESC
-  useEffect(function() {
-    const dialog = dialogRef.current;
-
-    if (dialog && hasNativeModalSupport) {
-      /* eslint-disable @typescript-eslint/no-non-null-assertion */
-      dialog.addEventListener('cancel', onCancel!);
-      return () => { dialog.removeEventListener('cancel', onCancel!); };
-      /* eslint-enable @typescript-eslint/no-non-null-assertion */
-    }
-    else {
-      return undefined;
-    }
-  }, [onCancel]);
+    'nx-modal--wide': variant === 'wide',
+    'nx-modal--narrow': variant === 'narrow'
+  });
 
   const ariaLabelAttrNames = ['aria-label', 'aria-labelledby'] as const,
       ariaLabels = pick(ariaLabelAttrNames, attrs),
-      attrsWithoutLabels = omit(ariaLabelAttrNames, attrs);
+      attrsWithoutLabels = omit([...ariaLabelAttrNames], attrs);
```

Comment:
What's the purpose of this change? `omit` does not mutate that array so I don't think there's a need to make a copy of it.

Reply:
> What's the purpose of this change? `omit` does not mutate that array so I don't think there's a need to make a copy of it.

Fixed here: 87e7cb6482292e40afaeac3721883024ea82027a